### PR TITLE
Complete pt translation

### PR DESCRIPTION
**Pull request by chicodurden21**
Portuguese version updated, portuguese native here. Everything was translated based on the formal you.

### CODE REVIEW

File: ghost/i18n/locales/pt/portal.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "(save {{highestYearlyDiscount}}%)": "",
+    "(save {{highestYearlyDiscount}}%)": "Poupe {{highestYearlyDiscount}}%",
```
Comment (sagzy) [inline ghost/i18n/locales/pt/portal.json:2]: Missing parenthesis `()` and no capital letter on P

File: ghost/i18n/locales/pt/portal.json [hunk 2]
```diff
@@ -49,125 +49,125 @@
...
     "In the event a permanent failure is received when attempting to send a newsletter, emails will be disabled on the account.": "No caso de uma falha permanente ser recebida ao tentar enviar uma newsletter, os emails serão desativados na conta.",
-    "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "No seu cliente de email, adicione {{senderEmail}} à sua lista de contactos. Isso indica ao seu fornecedor de email que os emails enviados a partir deste endereço devem ser confiáveis.",
-    "Invalid email address": "",
-    "Jamie Larson": "",
-    "jamie@example.com": "",
+    "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "No seu email, adicione {{senderEmail}} à sua lista de contactos. Esta ação indica ao seu fornecedor de email que os emails enviados a partir deste endereço devem ser confiáveis.",
+    "Invalid email address": "Email inválido",
+    "Jamie Larson": "Jane Doe",
+    "jamie@example.com": "janedoe@themoneygame.pt",
```
Comment (sagzy) [inline ghost/i18n/locales/pt/portal.json:80]: Could we leave the default here? Jamie Larson, and jamie@example.com

**Review by sagzy — CHANGES_REQUESTED** [review]
Thanks for the PR, great work! I left two comments. Additionally, the CI is failing due to a syntax error in `ghost/i18n/locales/pt/portal.json`

File: ghost/i18n/locales/pt/portal.json [hunk 3]
```diff
@@ -49,125 +49,125 @@
...
+    "There was a problem submitting your feedback. Please try again a little later.": "Houve um problema ao enviar sua avaliação. Tente novamente mais tarde por favor.",
+    "There was an error processing your payment. Please try again.": "Houve um problema ao processar o seu pagamento. Tente novamente por favor.",
+    "This site is invite-only, contact the owner for access.": "O acesso a este site é feito apenas por convite. Entre em contacto com o proprietário para obter acesso.",
+    "This site is not accepting payments at the moment.": "Este site não está a aceitar pagamentos de momento",
+    "To complete signup, click the confirmation link in your inbox. If it doesn't arrive within 3 minutes, check your spam folder!": "Para completar o registo, clique no link de confirmação enviado para a sua caixa de entrada. Se não o receber dentro de 3 minutos, verifique a sua pasta de spam!",
+    "To continue to stay up to date, subscribe to {{publication}} below.": "Para continuar atualizado, subscreva o {{publication}} aqui."
```
Comment (sagzy) [inline ghost/i18n/locales/pt/portal.json:142]: Syntax error: missing `,` at the end of the line

**Comment by chicodurden21** [comment]
done, I think you can merge it now. let me know if I need to change something else.

**Review by sagzy — APPROVED** [review]
Thanks for the updates! Merging :)

**Comment by chicodurden21** [comment]
the money game

Good afternoon, 

looks like the code hasn´t been updated.

Cheers

**Comment by sagzy** [comment]
> the money game
> 
> Good afternoon,
> 
> looks like the code hasn´t been updated.
> 
> Cheers

Hey @chicodurden21! :) The translations of this PR have been released in [Ghost v5.94.2](https://github.com/TryGhost/Ghost/releases/tag/v5.94.2).

Portal translations are still in beta and need to be enabled in Settings -> Labs -> Portal translation ([docs](https://ghost.org/docs/faq/translation/)). Is that enabled on your site?

**Comment by chicodurden21** [comment]
I´ve enabled the portal translations in the settings, however the words are still shown in English. 

By the way, how do I update to the latest ghost version?

**Comment by sagzy** [comment]
> I´ve enabled the portal translations in the settings, however the words are still shown in English.
> 
> By the way, how do I update to the latest ghost version?

@chicodurden21 👋 

If you're a Ghost (Pro) customer, your site is already updated automatically to the latest version. If you're a self-hoster, here are the [docs](https://ghost.org/docs/update/) on how to update Ghost to the latest version, using Ghost CLI

**Comment by chicodurden21** [comment]
there´s something wrong here then, because the sign in and subscribe button are still being shown in english. I am gonna contact the support. Thanks for the help anyway.

**Pull request merged**